test/inspector/test-inspector.js

```diff
@@ -199,6 +199,36 @@ function testI18NCharacters(session) {
   ]);
 }
 
+function testCommandLineAPI(session) {
+  session.sendInspectorCommands([
+    [ // we can use `require` outside of a callframe with require in scope
+      {
+        'method': 'Runtime.evaluate', 'params': {
+          'expression': 'typeof require("fs").readFile === "function"',
+          'includeCommandLineAPI': true
+        }
+      }, (message) => assert.strictEqual(true, message['result']['value'])
+    ],
+    [ // the global require does not have require.cache
+      {
+        'method': 'Runtime.evaluate', 'params': {
+          'expression': 'require.cache === undefined',
+          'includeCommandLineAPI': true
+        }
+      }, (message) => assert.strictEqual(true, message['result']['value'])
+    ],
+    [ // the `require` in the module shadows the command line API's `require`
+      {
+        'method': 'Debugger.evaluateOnCallFrame', 'params': {
+          'callFrameId': '{"ordinal":0,"injectedScriptId":1}',
+          'expression': 'typeof require.cache',
+          'includeCommandLineAPI': true
+        }
+      }, (message) => assert.strictEqual('object', message['result']['value'])
+    ],
+  ]);
+}
```

Comment:
I'll try. I'm not sure if `require.cache` is exposed globally by this PR. Since the require goes through the normal `module.require`, is there any reason to believe that deleting the require cache won't affect it?